README.md

```diff
@@ -266,7 +266,7 @@ a git repo that is not on the public Internet), you can mount in the host certif
 using the `COREOS_ASSEMBLER_CONTAINER_RUNTIME_ARGS` variable.
 
 ```
-$ export COREOS_ASSEMBLER_CONTAINER_RUNTIME_ARGS='-v /etc/pki:/etc/pki:ro'
+$ export COREOS_ASSEMBLER_CONTAINER_RUNTIME_ARGS='-v /etc/pki/ca-trust:/etc/pki/ca-trust:ro'
```

Comment:
IME this doesn't work in unprivileged mode though due to SELinux.

I personally like the model of `coreos-assembler shell` and doing everything inside there.  That way one can manually provision the cert e.g. and reuse it across multiple commands.

Reply:
> IME this doesn't work in unprivileged mode though due to SELinux.

You may be right since I only had this problem pop up when trying to build EL7 which has required `--privileged` so that's the thin use case I've tested in the past.